cpp/tests/binaryop/binop-compiled-test.cpp

```diff
@@ -27,6 +27,7 @@
 #include <cudf_test/type_lists.hpp>
 
 #include <cudf/utilities/error.hpp>
+#include <limits>
```

Comment:
```suggestion
```
This should be replaced right before `#include <type_traits>` without a blank line. They both belong to the STL header group.

Reply:
I wonder why clang-format did not do this for me (or at least complain)

Reply:
Ah, it only sorts in blocks.